src/cdk-experimental/menu/menu-bar.spec.ts

```diff
@@ -452,6 +452,27 @@ describe('MenuBar', () => {
 
           expect(document.activeElement).toEqual(fileMenuNativeItems[1]);
         }));
+
+        it('should handle keyboard actions if initial menu is opened programmatically', () => {
+          fixture.debugElement
+            .queryAll(By.directive(CdkMenuItem))[0]
+            .injector.get(CdkMenuItem)
+            .getMenuTrigger()!
+            .openMenu();
```

Comment:
nit: +4 indent for line overflow, so this would be e.g.
```suggestion
          fixture.debugElement
              .queryAll(By.directive(CdkMenuItem))[0]
              .injector.get(CdkMenuItem)
              .getMenuTrigger()!
              .openMenu();
```